src/events/interactionCreate.ts

```diff
@@ -266,16 +263,16 @@ async function handleAutocompleteInteraction(interaction: AutocompleteInteractio
  */
 export async function sendErrorMessage(
 	interaction: CommandInteraction | ButtonInteraction,
-	error: unknown
+	error_: unknown
 ): Promise<void> {
-	const errorMessage = toString(error);
+	const errorMessage = error_ instanceof Error ? error_.message : String(error_);
```

Comment:
`String()` would turn the object into `'[object Object]'`, whereas `JSON.stringify` would traverse and print the object's properties:

```suggestion
	const errorMessage = error_ instanceof Error ? error_.message : JSON.stringify(error_);
```